src/main/java/org/folio/circulation/infrastructure/storage/requests/RequestQueueRepository.java

```diff
@@ -38,11 +40,17 @@ public static RequestQueueRepository using(Clients clients) {
     return new RequestQueueRepository(RequestRepository.using(clients));
   }
 
-  public CompletableFuture<Result<LoanAndRelatedRecords>> get(
-    LoanAndRelatedRecords loanAndRelatedRecords) {
+  public CompletableFuture<Result<LoanAndRelatedRecords>> get(LoanAndRelatedRecords records) {
```

Comment:
Check what changeDueDate does to the request queue

Reply:
Created a separate ticket for TLR during due date change:
https://issues.folio.org/browse/CIRC-1362